snap/docker-build.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
 
-docker run --rm -v $PWD:/root/snap -w /root/snap -e SNAPCRAFT_SETUP_CORE=1\
-    snapcraft/xenial-amd64 make "$@"
+docker run --rm -v "$PWD":/root/snap -w /root/snap \
+       -e SNAPCRAFT_SETUP_CORE=1 \
+       -e KUBE_ARCH=amd64 \
+       -e KUBE_VERSION=$(curl -L https://dl.k8s.io/release/stable.txt) \
+       -v /usr/bin/curl:/usr/bin/curl \
```

Comment:
I'm sure the build system has all this stuff already, but to make it portable, perhaps this scripts should do something like:
```bash
if [ ! -e /usr/bin/[curl|make|etc] ]; then
  apt install [curl|make|etc]
fi
```

Reply:
I have a ansible playbook when auto creating jenkins slaves, ill add this there.